src/content/blog/whats-new-october.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "What's new in Astro - October"
+description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
+publishDate: "October 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/astro-3/blog-hero-post-header.webp"
+socialImage: "/src/content/blog/_images/astro-3/blog-social.webp"
+lang: "en"
+---
+
+## Astro releases
+
+- Astro `3.2`
+  - Control the browser history stack
+  - Trigger page navigation via JavaScript
+  - Route announcer for screen readers
+  - Dynamically add integrations
+  - [Read more about it here](/blog/astro-320/)
+- Astro `3.3`
+  - experimental `<Picture />` component
+  - Syntax highlighting improvements
+  - Package Provenance
+  - Quality of Life improvements
+  - [read more about it here](/blog/astro-330/)
+- Astro `3.4`
+  - Page Partials
+  - Image optimisation performance
+  - Experimental dev overlay
+  - [read more about it here](/blog/astro-340/)
+
+## Hacktoberfest
+
+[Hacktober](https://hacktoberfest.com/) is officially over now! Thank you all for your amazing contributions and PRs! Here are some of our statistics:
+
+- 2 Astro core PRs
+- 49 Starlight PRs
+- 62 Docs PRs
+
+## Docs
+
+We extended the tutorial with some requested features!
+
+- [Content Collections](https://docs.astro.build/en/tutorials/add-content-collections/)
+- [View Transitions](https://docs.astro.build/en/tutorials/add-view-transitions/)
+
+The Astro Docs team is also working on a "Rendering Modes" page, it's still in the works and we'd love your feedback on [the open PR](https://github.com/withastro/docs/pull/4667)
```

Comment:
```suggestion
"Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump in a shared CodeSpace or HackMD document and everyone can participate. This week's activities included:

- Live reviewing and testing the two new mini-tutorials
- Reviewing brand new content for the docs
- Improved our docs PR template and created a new GitHub action to welcome first-time contributors
- Using (and improving) the Astro Docs checklist for reviewing translation PRs when you don't speak the language
```
I think T & D is maybe more interesting as being representative of "what happened" that people wouldn't otherwise know about!